app-vcs/lazygit/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=lazygit
+PKGSEC=vcs
+PKGDES="Simple terminal UI for git commands"
+PKGDEP="git"
+BUILDDEP="go"
+ABINSTALL=lazygit
```

Comment:
Why is this required?

```suggestion
```

Reply:
This is not necessary and has been deleted